src/libsystemd-network/dhcp6-internal.h

```diff
@@ -14,6 +14,16 @@
 #include "macro.h"
 #include "sparse-endian.h"
 
+typedef struct sd_dhcp6_option {
+        unsigned n_ref;
+
+        uint16_t option;
+        void *data;
+        size_t length;
+} sd_dhcp6_option;
```

Comment:
It doesn't make sense to put a publicly-prefixed identifier (with `sd_`) in a private header. But I see that `sd_dhcp_option` is defined like that too. OK, so let's leave this. We'll have to move them when we finally get around to making those interfaces public.